app/scripts/preload.js

```diff
@@ -287,7 +287,7 @@ try {
 		}
 	});
 } catch(e) {
-	console.log('Spellchecker unavailable');
+	console.log('spellchecker module unavailable');
```

Comment:
this also has the added benefit of making this PR seven billions times moar better than this PR https://github.com/RocketChat/Rocket.Chat.Electron/pull/202